odk1-src/form-audit-log.rst

```diff
@@ -90,6 +90,22 @@ You can enable change tracking so that old answers and new answers will be added
 
   audit, audit, track-changes=true
 
+Tracking reason for edit
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 1.25
+
+  `ODK Collect v1.25.0 <https://github.com/opendatakit/collect/releases/tag/v1.25.0>`_
+
+You can add to :tc:`track-changes-reason=on-form-edit` to prompt enumerators to enter a reason when they save changes to a form:
```

Comment:
Should be `track-changes-reasons` with an `s` at the end!

Reply:
Ach! Every time.